source/Cosmos.System2/Keyboard/ScanMaps/USDvorakLayout.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cosmos.HAL;
+
+namespace Cosmos.System.ScanMaps
+{
+    /// <summary>
+    /// US_Dvorak class. Represent US_Dvorak keyboard layout.
+    /// </summary>
+    public class US_Dvorak : ScanMapBase
+    {
+        /// <summary>
+        /// Create new instance of the <see cref="US_Dvorak"/> class.
+        /// </summary>
+        public US_Dvorak()
+        {
+        }
+
+        /// <summary>
+        /// Init key list.
+        /// </summary>
+        protected override void InitKeys()
+        {
+            _keys = new List<KeyMapping>();
+
+            #region Keys
+
+            /*     Scan  Norm Shift Ctrl Alt     Num  Caps ShCaps ShNum ConsoleKeyEx */
+            _keys.Add(new KeyMapping(0x00, ConsoleKeyEx.NoName));
+            _keys.Add(new KeyMapping(0x01, ConsoleKeyEx.Escape));
+            /* 1 -> 9 */
+            _keys.Add(new KeyMapping(0x02, '1', '!', '1', '1', '!', '1', ConsoleKeyEx.D1));
+            _keys.Add(new KeyMapping(0x03, '2', '@', '2', '2', '@', '2', ConsoleKeyEx.D2));
+            _keys.Add(new KeyMapping(0x04, '3', '#', '3', '3', '#', '3', ConsoleKeyEx.D3));
+            _keys.Add(new KeyMapping(0x05, '4', '$', '4', '4', '$', '4', ConsoleKeyEx.D4));
+            _keys.Add(new KeyMapping(0x06, '5', '%', '5', '5', '%', '5', ConsoleKeyEx.D5));
+            _keys.Add(new KeyMapping(0x07, '6', '^', '6', '6', '^', '6', ConsoleKeyEx.D6));
+            _keys.Add(new KeyMapping(0x08, '7', '&', '7', '7', '&', '7', ConsoleKeyEx.D7));
+            _keys.Add(new KeyMapping(0x09, '8', '*', '8', '8', '*', '8', ConsoleKeyEx.D8));
+            _keys.Add(new KeyMapping(0x0A, '9', '(', '9', '9', '(', '9', ConsoleKeyEx.D9));
+            _keys.Add(new KeyMapping(0x0B, '0', ')', '0', '0', ')', '0', ConsoleKeyEx.D0));
+            /* -, =, Bksp, Tab */
+            _keys.Add(new KeyMapping(0x0C, '[', '{', '[', '{', '[', '{', ConsoleKeyEx.LBracket));
+            _keys.Add(new KeyMapping(0x0D, ']', '}', ']', '}', ']', '}', ConsoleKeyEx.RBracket));
+            //_keys.Add(new KeyMapping(0x0C, '-', '_', '-', '-', '_', '-', ConsoleKeyEx.Minus));
+            //_keys.Add(new KeyMapping(0x0D, '=', '+', '=', '=', '+', '=', ConsoleKeyEx.Equal));
+            _keys.Add(new KeyMapping(0x0E, ConsoleKeyEx.Backspace));
+            _keys.Add(new KeyMapping(0x0F, ConsoleKeyEx.Tab));
+            /*      QWERTYUIOP[] */
+            _keys.Add(new KeyMapping(0x10, '\'', '"', '\'', '\'', '"', '"', ConsoleKeyEx.Apostrophe)); //'q', 'Q', 'q', 'Q', 'q', 'Q', ConsoleKeyEx.Q));
```

Comment:
Could you please remove the commented out code in the following lines?

Reply:
done :)